test/crypto/sign.js

```diff
@@ -315,5 +315,26 @@ ${defaultSignature}
 				(multiSignature).should.be.eql(expectedMultiSignature);
 			});
 		});
+
+		describe('encrypting passphrase', () => {
+			const secretPassphrase = 'minute omit local rare sword knee banner pair rib museum shadow juice';
+			const password = 'myTotal53cr3t%&';
+			const encryptString = cryptoModule.aesEncrypt(secretPassphrase, password);
+
+			describe('#aesEncrypt', () => {
+				it('should encrypt a given secret with a password', () => {
+					(encryptString).should.be.ok();
+					(encryptString).should.be.type('string');
+					(encryptString).should.containEql('$');
+				});
+			});
+
+			describe('#aesDecrypt', () => {
+				it('should decrypt a given cipher with a password', () => {
+					const decryptedString = cryptoModule.aesDecrypt(encryptString, password);
+					(decryptedString).should.be.eql(secretPassphrase);
+				});
+			});
```

Comment:
These tests are not atomic. (Although an additional integration test would make sense.)